### PR TITLE
DirListing: Make Renderer's private method protected

### PR DESCRIPTION
**Pull request by martinRenou**
This makes subclassing nicer

### CODE REVIEW

File: packages/filebrowser/src/listing.ts [hunk 1]
```diff
@@ -2111,7 +2111,7 @@ export namespace DirListing {
     /**
      * Create a node for a header item.
      */
-    private _createHeaderItemNode(label: string): HTMLElement {
+    protected _createHeaderItemNode(label: string): HTMLElement {
```
Comment (afshin) [inline packages/filebrowser/src/listing.ts:2114]: If we decide to make this `protected`, we should remove the `_` from the name; that is strictly for `private` methods and attributes.

**Review by afshin — APPROVED** [review]
:+1:

File: packages/filebrowser/src/listing.ts (hunk 1 above)
Comment (martinRenou) [reply packages/filebrowser/src/listing.ts]: Thanks for the review!

**Comment by blink1073** [comment]
Currently making an alpha release, will hold off on merging this.

**Pull request merged**